website/templates/project/register.mako

```diff
@@ -3,40 +3,51 @@
 
 <legend class="text-center">Register</legend>
 
-% if schema:
-    <%include file="metadata/register_${ str(metadata_version) }.mako" />
+% if node.get('registered_schema'):
+  <div id="registrationMetaDataScope" class="container scripted">
+    <div class="row">
+      <div class="span8 col-md-2 columns eight large-8">        
+        <ul class="nav nav-stacked list-group" data-bind="foreach: {data: metaSchema.schema.pages, as: 'page'}">
+          <li class="re-navbar">
+            <a class="registration-editor-page" id="top-nav" style="text-align: left; font-weight:bold;" data-bind="text: title, attr.href: '#' + page.id">
```

Comment:
I don't see this defined in registrations.css. (`registration-editor-page`) At this point there's enough style questions in the template that I'll stop noting them individually, pending in-person conversation. We can revisit further if this remains a thing.